deltachat-ios/Controller/Settings/Proxy/ProxyTableViewCell.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+import DcCore
+
+class ProxyTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = "ProxyTableViewCell"
+
+    let hostLabel: UILabel
+    let protocolLabel: UILabel
+    let connectionLabel: UILabel
+
+    private let contentStackView: UIStackView
+    private let subtitleStackView: UIStackView
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        hostLabel = UILabel()
+        protocolLabel = UILabel()
+
+        let detailsColor: UIColor
+        if #available(iOS 13.0, *) {
+            detailsColor = .secondaryLabel
+        } else {
+            detailsColor = .darkGray
+        }
+        protocolLabel.textColor = detailsColor
+        protocolLabel.font = UIFont.preferredFont(forTextStyle: .footnote)
+
+        connectionLabel = UILabel()
+        connectionLabel.textColor = detailsColor
+        connectionLabel.font = UIFont.preferredFont(forTextStyle: .footnote)
+
+        subtitleStackView = UIStackView(
+            arrangedSubviews: [
+                UIView.borderedView(around: protocolLabel, borderWidth: 1, borderColor: detailsColor, cornerRadius: 2, padding: NSDirectionalEdgeInsets(top: 2, leading: 2, bottom: 2, trailing: 2)),
+                connectionLabel,
+                UIView()
+            ]
+        )
+        subtitleStackView.translatesAutoresizingMaskIntoConstraints = false
+        subtitleStackView.spacing = 4
+
+        contentStackView = UIStackView(arrangedSubviews: [hostLabel, subtitleStackView])
+        contentStackView.translatesAutoresizingMaskIntoConstraints = false
+        contentStackView.axis = .vertical
+        contentStackView.spacing = 6
+
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        contentView.addSubview(contentStackView)
+
+        setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+
+    private func setupConstraints() {
+        let constraints = [
+            contentStackView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 8),
+            contentStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 8),
```

Comment:
design: use whitespace aligning with the other elements on the left - as well as having the same whitespace left/right

```suggestion
            contentStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
```

<details>
<summary>before / after</summary>

<img width=320 src=https://github.com/user-attachments/assets/6c958e33-817c-4c42-8216-11e451b881b9> &nbsp;  &nbsp;  &nbsp;  <img width=320 src=https://github.com/user-attachments/assets/aaebbe5e-f274-49bc-9ffe-574ee3a50818>


(including the tag's whitespace changes)

</details>